zulipterminal/cli/run.py

```diff
@@ -5,11 +5,15 @@
 import tempfile
 from typing import Dict, Any, List, Optional
 from os import path, remove
+import logging
 
 from zulipterminal.core import Controller
 from zulipterminal.model import ServerConnectionFailure
 from zulipterminal.config.themes import THEMES
 
+LOG_FILENAME = 'debug.log'
```

Comment:
We can improve upon this further in future, but for now what do you think about having this be something more specific such as `zulip-terminal-tracebacks.log` instead? The file will be placed in the current directory, which could be anywhere, so `debug.log` would not be as clear to end-users on its own? Also `debug.log` is used for actual debugging during development so keeping them distinct could be useful?